feedhenry-raincatcher/raincatcher-core
# Security configuration with redis support.

## Motivation

Add redis to hold session.

## Progress
- [x] Moved critical passport.js JWT token implementation that was 
interacting with session config (required for documentation)
- [x] Moved to use users as JWT payload (minor change with huge production implications)
- [x] Refactored config - to improve our documentation (minimize sets of steps for keycloak)
- [x] Included redis session into the code. 
- [x] Configured session to be based on redis.
- [x] Improve coverage and unit tests spies.

## Additional Notes

Steps to verify:

Keycloak and passport separately
1. Login to portal application
2. Restart server 
3. Go back. Session should still be valid and user should be logged in.
